Intro_to_objective-c/NSString+NSString_Category.m

```diff
@@ -0,0 +1,36 @@
+//
+//  NSString+NSString_Category.m
+//  Intro_to_objective-c
+//
+//  Created by Luay Younus on 4/17/17.
+//  Copyright © 2017 Luay Younus. All rights reserved.
+//
+
+#import "NSString+NSString_Category.h"
+
+@implementation NSString (NSString_Category)
+
+NSString *seperator = @" ";
+
+-(NSArray *) stringArray{
+    NSArray *words = [self componentsSeparatedByString: seperator];
+    return words;
+}
+
++(NSString *) reverseStringMethod : (NSString *) myString{
+    
+    NSString *result = [NSMutableString stringWithString: @""];
```

Comment:
If you just need a new, empty mutable string, use `[[NSMutableString alloc] init];`